hello/admin.py

```diff
@@ -26,3 +26,14 @@ class StudentAdmin(admin.ModelAdmin):
 class DealerAdmin(admin.ModelAdmin):
     list_display = ['name', 'niche', 'sale']
 
+@admin.register(Quality)
+class QualityAdmin(admin.ModelAdmin):
+    model = Quality
+
+    fiels = ['quality', 'is_good']
```

Comment:
Have you tested this code? Is it working as expected? What about action on single object?